src/routes/index.js

```diff
@@ -89,6 +89,52 @@ const getDevice = (device) => {
   });
 }
 
+async function queryGitHubReleasesApi(requestedInstallerPackage) {
+  return new Promise ((resolve, reject) => {
+    request.get({
+      url: "https://api.github.com/repos/ubports/ubports-installer/releases/latest",
+      json: true,
+      headers: { 'User-Agent': 'request' }
+    },
+    (err, res, bod) => {
+      if (!err && res.statusCode === 200) {
+        for (asset in bod.assets) {
+          if(bod.assets[asset].browser_download_url.split('.').pop().toLowerCase() == requestedInstallerPackage)
```

Comment:
would be better to use endswith() here then split.pop

Reply:
good idea, thanks